white-paper.md

```diff
@@ -159,39 +159,39 @@ This design is driven by the hypothesis that each dependency is itself a package
 ![Dependencies analysis diagram.](img/figure-3.svg){#fig:dep-analysis}
 
 
-In @fig:dep-analysis, the submission of package A triggers an analysis of runtime dependencies 1 through n and build dependencies 1 through n, while runtime dependencies 1.1 through 1.n and build dependencies 1.1 through 1.n were analyzed when package B was submitted. We will apply the same methodology for incentive distribution as the steeped tokens are distributed across all dependencies, thus recursively steeping the packages listed as dependencies (see @fig:steeping-rewards).
+In @fig:dep-analysis, the submission of package A triggers an analysis of runtime dependencies 1 through n and build dependencies 1 through n, while runtime dependencies 1.1 through 1.n and build dependencies 1.1 through 1.n were analyzed when package B was submitted. We will apply the same methodology for incentive distribution as the steeped tokens are distributed across all dependencies, thus recursively steeping the packages listed as dependencies (See @fig:steeping-rewards).
 
 ![Steeping rewards distribution across dependencies.](img/figure-2.svg){#fig:steeping-rewards}
 
 
-Versioning and conflicting dependencies are significant challenges, and troubleshooting them can turn into massive time drains. To address this, we propose each package be subject to a full dependency scan upon submission so we can ensure that the package complies with the following rules for semantic version ranges.
+Versioning and conflicting dependencies are significant challenges, and troubleshooting them can turn into massive time drains. To address this, we propose each package be subject to a comprehensive dependency scan upon submission, so we can ensure that the package complies with the following rules for semantic version ranges:
 
-* Packages may only depend on major versions or greater than specific versions, but for the whole range.
+* Packages may only depend on a major version or any version preceding the next major version increment (e.g., >= 5.2 < 6).
```

Comment:
@mxcl 
Perhaps this is clearer? This is really challenging language. 😅

Reply:
It’s an “and”, I suggest:

> Packages may only constrain their dependence to a major version though the starting range can be any semantic version (e.g., >=5.2.1 <6).

Reply:
Before I make the edit, are you saying "dependence" or "dependencies"?

This?
> Packages may only constrain their _dependencies_ to a major version, though the start of the range can be any semantic version (e.g., >=5.2.1 <6).

Reply:
I did mean *dependence* but I guess *dependencies* is better.